src/Nancy.Swagger/Standalone.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Nancy.Swagger.Services;
+using Nancy.TinyIoc;
+using Swagger.ObjectModel;
+
+namespace Nancy.Swagger
+{
+    public static class Standalone
+    {
+        public static SwaggerRoot Generate(TinyIoCContainer container, NancyContext context)
```

Comment:
But then this class is useless, since you could just call the provider.GetSwaggerJson in the calling scope. 

I don't think this warrants a new class. You should just call GetSwaggerJson in the demo.